src/Core/FileSystemModule/IETempFile.cpp

```diff
@@ -0,0 +1,3 @@
+//
```

Comment:
This entire comment should go. We don't do comments like this for now. Furthermore I question the validity of including a file that we don't use.